functions/Get-DbaDatabase.ps1

```diff
@@ -159,58 +165,104 @@
 		[datetime]$NoLogBackupSince,
 		[switch]$Silent
 	)
-
+	
 	begin {
-
-		if ($NoUserDb -and $NoSystemDb) {
-			Stop-Function -Message "You cannot specify both NoUserDb and NoSystemDb" -Continue -Silent $Silent
+		
+		if ($ExcludeUserDb -and $ExcludeSystemDb) {
+			Stop-Function -Message "You cannot specify both ExcludeUserDb and ExcludeSystemDb" -Continue -Silent $Silent
 		}
-
+		
 	}
 	process {
 		if (Test-FunctionInterrupt) { return }
-
+		
 		foreach ($instance in $SqlInstance) {
 			try {
-			    Write-Message -Level Verbose -Message "Connecting to $instance"
+				Write-Message -Level Verbose -Message "Connecting to $instance"
 				$server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
 			}
 			catch {
 				Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
 			}
-
-			if ($NoUserDb) {
-                $DBType = @($true)
+			
+			## Get last used information from the DMV
+			$querylastused = "WITH agg AS
+				(
+				  SELECT 
+				       max(last_user_seek) last_user_seek,
+				       max(last_user_scan) last_user_scan,
+				       max(last_user_lookup) last_user_lookup,
+				       max(last_user_update) last_user_update,
+				       sd.name dbname
+				   FROM
+				       sys.dm_db_index_usage_stats, master..sysdatabases sd
+				   WHERE
+				     database_id = sd.dbid AND database_id > 4
+					  group by sd.name 
+				)
+				SELECT 
+				   dbname,
+				   last_read = MAX(last_read),
+				   last_write = MAX(last_write)
+				FROM
+				(
+				   SELECT dbname, last_user_seek, NULL FROM agg
+				   UNION ALL
+				   SELECT dbname, last_user_scan, NULL FROM agg
+				   UNION ALL
+				   SELECT dbname, last_user_lookup, NULL FROM agg
+				   UNION ALL
+				   SELECT dbname, NULL, last_user_update FROM agg
+				) AS x (dbname, last_read, last_write)
+				GROUP BY
+				   dbname
+				ORDER BY 1;"
+			# put a function around this to enable Pester Testing and also to ease any future changes
+			function Invoke-QueryDBlastUsed {
+				try {
+					$server.Query($querylastused)
```

Comment:
Fred added some awesome methods to $server and $database! So now we're using $server.Query() which outputs the tables thing like you were doing